lib/cachex/warmer.ex

```diff
@@ -99,18 +121,18 @@ defmodule Cachex.Warmer do
 
           # set pairs without options
           {:ok, pairs} ->
-            Cachex.put_many(cache, pairs)
+            Cachex.put_many(name, pairs)
 
           # set pairs with options
           {:ok, pairs, options} ->
-            Cachex.put_many(cache, pairs, options)
+            Cachex.put_many(name, pairs, options)
         end
 
         # trigger the warming to happen again after the interval
-        :erlang.send_after(interval(), self(), :cachex_warmer)
+        timer_ref = :erlang.send_after(interval(), self(), :cachex_warmer)
```

Comment:
Probably just call it `timer`, if you end up changing anything in this file again - no worries if not.

Reply:
Updated